WordPress/Docs/CodeAnalysis/PerfomantPHPFunctionsStandard.xml

```diff
@@ -0,0 +1,22 @@
+<documentation title="Prefer Performant PHP">
+    <standard>
+        <![CDATA[
+    Various PHP functions are legacy and less performant than other language constructs.
+    This sniff will help identify use of functions that may be better served using other options.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Using native type casting.">
+            <![CDATA[
+$var = '100'; // string.
```

Comment:
For a more realistic/more relatable example, you may want to use a function in the example which has multiple return types. (in both code samples)

```php
$var = fn_returning_int_or_false();
```